Source/WebGPU/WebGPU/CommandEncoder.mm

```diff
@@ -738,6 +679,8 @@ static bool isRenderableTextureView(const auto& texture)
                 texture.setPreviouslyCleared();
             if (attachment.resolveTarget)
                 protectedFromAPI(attachment.resolveTarget)->setPreviouslyCleared();
+            if (attachment.resolveTexture)
+                protectedFromAPI(attachment.resolveTexture)->setPreviouslyCleared();
```

Comment:
Is this a drive by fix for the C++ implementation? What's the impact of not doing this / should there be a test for this?

Reply:
Yes it is :) Noticed when making the corresponding Swift change - https://github.com/WebKit/WebKit/pull/51686/files#r2400184367

The impact of not doing is previously written to texture contents would get zeroed during a partial update. 

The test should be added to https://gpuweb.github.io/cts/standalone/?q=webgpu:api,operation,resource_init,texture_zero:* but since we pass that one at 100% the CTS seems to be missing a case there

I filed https://bugs.webkit.org/show_bug.cgi?id=300053 to add that